### PR TITLE
(WIP) Appsembler theme for Analytics

### PR DESCRIPTION
**Pull request by OmarIthawi**
# Basic Branding

# TODO
 - [ ] Get it reviewed by Matej
 - [ ] Resize the logos to fit (after Matej reviews/updates it)
 - [ ] Get it reviewed by another engineer
 - [x] Finish the configuration repo PR for the rest of the branding (https://github.com/noderabbit-team/edx-configs/pull/167)

# How to Test

 - Clone this repo `git@github.com:appsembler/edx-analytics-dashboard.git`
 - Follow the installation guide in the [Ficus README.md](https://github.com/appsembler/edx-analytics-dashboard/tree/open-release/ficus.master). The steps are mainly:
   - `$ mkvirtualenv`
   - `$ make develop`
   - `$ make migrate`
 - Run the following command: `$ THEME_CSS=sass/themes/appsembler.scss ./manage.py runserver 0.0.0.0:9000`

**Hint:** Try `edx.scss`, `open-edx.scss` and `appsembler.scss` to see the differences.

## What to Test
 - Open the homeapge: `http://localhost:9000/` (notice the new logo and the aligned login button)
 - Open the authentication error page `http://localhost:9000/403/` (notice the negative logo)

## What's not to Test
 - The email, since it's configurable using Ansible, so a PR should be created in the https://github.com/noderabbit-team/edx-configs/ repo.
 - The company name (Open edX vs. Appsembler), also configurable using Ansible.

### CODE REVIEW

**Comment by OmarIthawi** [comment]
[Insights: change "Contact edX support" to something else](https://trello.com/c/4iJkiPIv/1289-insights-change-contact-edx-support-to-something-else)

**Comment by OmarIthawi** [comment]
@grozdanowski your input is appreciated here :)

File: analytics_dashboard/settings/base.py [hunk 1]
```diff
@@ -416,7 +416,7 @@
 
 ########## THEME CONFIGURATION
 # Path of the SCSS file to use for the site's theme
-THEME_SCSS = 'sass/themes/open-edx.scss'
+THEME_SCSS = os.getenv('THEME_SCSS', 'sass/themes/open-edx.scss')
```
Comment (melvinsoft) [inline analytics_dashboard/settings/base.py:419]: @OmarIthawi We don't need this change, the configuration playbooks are including the `INSIGHTS_THEME_SCSS` in the insights yaml config (`/edx/etc/insights.yml`). https://github.com/appsembler/configuration/blob/appsembler/ficus/master/playbooks/roles/insights/defaults/main.yml#L125

For more details, the production settings loads the env in this way:
```
from analytics_dashboard.settings.base import *
from analytics_dashboard.settings.yaml_config import *
from analytics_dashboard.settings.logger import get_logger_config
```
So `THEME_SCSS` is going to be replaced anyway.

**Review by melvinsoft — CHANGES_REQUESTED** [review]
@OmarIthawi Thanks for this PR, is reviewed, I've requested a change, please take a look!

File: analytics_dashboard/settings/base.py (hunk 1 above)
Comment (OmarIthawi) [reply analytics_dashboard/settings/base.py:419]: Yeah, we don't. I didn't see those imports before!

**Comment by OmarIthawi** [comment]
Thanks @melvinsoft, I'll figure out a better solution like you suggested!

**Comment by OmarIthawi** [comment]
Forking is bad, let's close this and move to ProdPad, more info in  [Insights: change "Contact edX support" to something else](https://trello.com/c/4iJkiPIv/1289-insights-change-contact-edx-support-to-something-else).

**Pull request closed without merging**